src/fundus/scraping/crawler.py

```diff
@@ -150,6 +150,7 @@ def crawl(
         only_complete: Union[bool, ExtractionFilter] = Requires("title", "body", "publishing_date"),
         url_filter: Optional[URLFilter] = None,
         only_unique: bool = True,
+        ignore_deprecated: bool = False,
```

Comment:
I would prefer to set the default to `True`, it doesn't hurt if the deprecated publishers cannot be crawled, but if they can and are excluded by default most won't notice. As an example, crawling Reuters works perfectly fine for me.